README.md

```diff
@@ -120,7 +120,7 @@ We recommend using a dedicated Google Service Account for each namespace. This r
 to setup (once per namespace).
 
 The Google Service Account running the CloudRun service will be mapped to a K8s namespace. Currently the 'default'
-K8s service account in the namespace is used - custom setup will be documented separately. [TODO is this documented?]
```

Comment:
There are comments in the code for the required env variables - but no separate doc or tests/examples.